test/test/jfr/JfrTests.java

```diff
@@ -78,6 +80,91 @@ public void parseMultiModeRecording(TestProcess p) throws Exception {
         Assert.isGreater(eventsCount.get("jdk.ObjectAllocationInNewTLAB"), 50);
     }
 
+    /**
+     * Test to validate profiling output with "--all" flag for Linux without event override
+     *
+     * @param p The test process to profile with.
+     * @throws Exception Any exception thrown during profiling JFR output parsing.
+     */
+    @Test(mainClass = JfrMutliModeProfiling.class, agentArgs = "start,all,file=%f.jfr", os = Os.LINUX)
+    @Test(mainClass = JfrMutliModeProfiling.class, agentArgs = "start,all,alloc=2m,file=%f.jfr", os = Os.LINUX)
+    public void allModeRecordingLinuxWithoutEventOverride(TestProcess p) throws Exception {
```

Comment:
Try to keep method names descriptive, but concise. Overly long identifiers are not readable and not very convenient for using in test filters.